README.md

```diff
@@ -11,108 +11,12 @@ This template includes setup for:
 - `docs`: a directory for project documentation
 - `infra`: a directory for common infrastructure
 
-## How to Run
+## Installation
 
-### Without Docker
```

Comment:
🤔  This got removed awhile ago, I think you might need to pull in the latest changes into this branch

Reply:
oh wow didn't realize i was working off of an outdated repo, thanks